### DISCUSSION TITLE
1.38.7b0 filter sharing information feature

### DISCUSSION THREAD

**Discussion by peace2000** [uBlockOrigin/uBlock-issues#1762] [Ideas] [answered]
Concerning this:

_A new Support pane has been added to the dashboard, to make it easy for users to share technical information about their current uBO's configuration, and consequently make it easier for volunteers to diagnose reported issues._

Could those filter list names made bold so that it's easier to quickly distinguish them from other info?

listset: 
      default: 
        **ublock-filters:** 
          filters used/total: 29885 / 29945
          updated last/next (days): -0.05 / 3.95
        **ublock-badware:** 
          filters used/total: 2819 / 2820
          updated last/next (days): -0.04 / 3.96
        **ublock-privacy:** 
          filters used/total: 171 / 171
          updated last/next (days): -0.04 / 3.96
        **ublock-abuse:** 
          filters used/total: 64 / 67
          updated last/next (days): -0.04 / 3.96
        **ublock-unbreak:** 
          filters used/total: 1694 / 1694
          updated last/next (days): -0.04 / 3.96
        **easylist:** 
          filters used/total: 58998 / 59514
          updated last/next (days): -0.04 / 3.96
        **easyprivacy:** 
          filters used/total: 24862 / 25493
          updated last/next (days): -0.04 / 3.96
          
         
Edit, ok in GH it doesn't make big difference but in some other platforms it could make bigger difference.

**Comment by gorhill** [comment]
The data is meant to render as code, because there might be special markdown characters, i.e.:

    browserFlavor: ublock webext devbuild firefox user_stylesheet html_filtering mozilla
    browserVersion: 95,
    extensionName: uBlock Origin
    extensionVersion: 1.38.7b0
    modifiedUserSettings: 
      advancedUserEnabled: true,
      popupPanelSections: 31
    modifiedHiddenSettings: 
      consoleLogLevel: info
      extensionUpdateForceReload: true
    listset: 
      default: 
        ublock-filters: 
          filters used/total: 29665 / 29698
          updated last/next (days): -3.32 / 0.68
        ublock-badware: 
          filters used/total: 2866 / 2867
          updated last/next (days): -2.03 / 1.97
        ublock-privacy: 
          filters used/total: 171 / 171
          updated last/next (days): -2.03 / 1.97
        ublock-abuse: 
          filters used/total: 70 / 70
          updated last/next (days): -2.02 / 1.98
        ublock-unbreak: 
          filters used/total: 1689 / 1689
          updated last/next (days): -0.09 / 3.91
        easylist: 
          filters used/total: 58955 / 59463
          updated last/next (days): -1.15 / 2.85
        easyprivacy: 
          filters used/total: 25275 / 25485
          updated last/next (days): -2.03 / 1.97
        plowe-0: 
          filters used/total: 3676 / 3678
          updated last/next (days): -7.66 / 5.34
        urlhaus-1: 
          filters used/total: 16645 / 16645
          updated last/next (days): -0.81 / 0.19
      added: 
        http://sbc.io/hosts/hosts: 
          filters used/total: 79542 / 92774
          updated last/next (days): -1.04 / 3.96
        adguard-spyware-url: 
          filters used/total: 306 / 306
          updated last/next (days): -0.09 / 3.91
        block-lan: 
          filters used/total: 43 / 43
          updated last/next (days): -5.49 / 24.51
    filterset: [array of 88 redacted]
    trustedset: 
      added: [array of 3 redacted]
    hostRuleset: 
      added: [array of 288 redacted]
      removed: 
        behind-the-scene * * noop
        behind-the-scene * image noop
        behind-the-scene * inline-script noop
        behind-the-scene * 1p-script noop
        behind-the-scene * 3p-script noop
    switchRuleset: 
      added: [array of 101 redacted]
      removed: 
        no-large-media: behind-the-scene false

It's not possible to style inside code block.

**Comment by peace2000** [comment]
Oh ok.

Would be it possible to use display names for those filter lists?

_urlhaus-1:_  sounds pretty cryptic, compared to:

![kuva](https://user-images.githubusercontent.com/17256841/137353432-31e56b68-3c74-4f97-a05e-c6d731173365.png)

Or is there a technical reason also why it's this way?

**Reply by gorhill** [reply, marked as answer]
I want to minimize redundant information, and I thought the long name took to much place. The token-based name makes it clear it's from a stock filter list and not an imported list, so it's shouldn't be difficult to see which stock filter list is being used.
